asyncauth.go

```diff
@@ -334,22 +347,22 @@ func (config *Config) downloadKubeConfig(w http.ResponseWriter, req *http.Reques
 }
 
 func (config *Config) renderKubeconfig(profileName string) ([]byte, error) {
-	parsed, _ := url.Parse(config.Clusters[0].Redirect_URI)
+	parsed, _ := url.Parse(config.getFirstClusterOrPanic().Redirect_URI)
 	appURL := fmt.Sprintf("%s://%s", parsed.Scheme, parsed.Host)
 	asyncAuthURL := fmt.Sprintf("%s%s", appURL, config.Web_Path_Prefix)
 
 	kUser := KConfigUser{
 		Name:    profileName,
 		AuthURL: asyncAuthURL,
-		Command: runPath,
```

Comment:
I didn't see `runPath` constant used anywhere else. Remove it?

Reply:
looks like it's still used here: https://github.com/mesosphere/dex-k8s-authenticator/blob/480d76031ccf7ea49c075163e062c7fd90c8154d/asyncauth.go#L238